main/lib/src/RunControl.cc

```diff
@@ -80,8 +82,12 @@ namespace eudaq {
     SendCommand("CONFIG", to_string(config));
     if (config.SetSection("RunControl")) {
       m_runsizelimit = config.Get("RunSizeLimit", 0LL);
+      m_runeventlimit = config.Get("RunEventLimit", 0LL);
+      m_nextconfigonfilelimit = config.Get("NextConfigFileOnFileLimit", false);
```

Comment:
@simonspa The feature does exactly what you guessed. Once the run is changed the next config file is picked. The range is from where you start the first run with this enable in the GUI until there are no more config files left (in alphabetical order). The actual changing is done in euRun.hh.